client/src/dojo/modelManager/ConfigManager.ts

```diff
@@ -509,6 +509,10 @@ export class ClientConfigManager {
     return EternumGlobalConfig.resources.resourcePrecision;
   }
 
+  getResourceMultiplier() {
+    return EternumGlobalConfig.resources.resourceMultiplier;
```

Comment:
this needs to be fetched from RECS not from the config

Reply:
resource multiplier isn't present in the state, it's a value we use to set the config